functions/check_prices/test_check_prices.py

```
@@ -0,0 +1,50 @@
+# 親ディレクトリをパスに追加
+sys.path.insert(0, str(Path(__file__).parent.parent))
+
+from check_prices.main import check_prices
+
+
+def test_check_prices_success():
+    """正常な価格チェック"""
+    request = Mock()
+    request.method = 'GET'
+    
+    response_body, status_code, headers = check_prices(request)
```

Ruff (0.14.2)

Unpacked variable `headers` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)